views/profileMessages.handlebars

```diff
@@ -0,0 +1,109 @@
+        </ul>
+      </div>
+    </div>
+  </div>
+</nav>
+
+<!-- This contains our entire profile -->
+<div class="container profileContainer">
+  <div class="row">
+
+<!-- This contains the menu  -->
+    <div class="col s3">
+       <div class="collection">
+         <a href="/profile/1" class="collection-item">Profile</a>
+         <a href="/profile/1/listings" class="collection-item ">Listings</a>
+         <a href="/profile/1/bids" class="collection-item">Bids</a>
+         <a href="/profile/1/messages" class="collection-item active">Messages</a>
+      </div>
+    </div>
+
+   <!-- Begin displaying Messages here -->
+   <div class="col s9">
+
+
+     <div class="row">
+       <div class="col s12 m12">
+        <div class="card horizontal hoverable">
+          <div class="messageImage valign-wrapper">
+            <img class="circle" src="http://placehold.it/120">
+          </div>
+          <div >
+            <div class="card-content">
+
+              <!-- Replace this with dynamic address data -->
+              <p>This is a sample message that came from this user. This may have a question or a response about the home</p>
+
+
+            </div>
+
+            <div class="card-content right-align">
+              <!-- Replace this with the number of bids that have been made -->
+              <p>9:43 am</p>
+            </div>
+          </div>
+        </div>
+      </div>
+     </div>
+
+     <div class="row">
+       <div class="col s12 m12">
+        <div class="card horizontal hoverable">
+          <div class="messageImage valign-wrapper">
+            <img class="circle" src="http://placehold.it/120">
+          </div>
+          <div >
+            <div class="card-content">
+
+              <!-- Replace this with dynamic address data -->
+              <p>This is a sample message that came from this user. This may have a question or a response about the home</p>
+
+
+            </div>
+
+            <div class="card-content right-align">
+              <!-- Replace this with the number of bids that have been made -->
+              <p>Thursday</p>
+            </div>
+          </div>
+        </div>
+      </div>
+     </div>
+
+     <div class="row">
+       <div class="col s12 m12">
+        <div class="card horizontal hoverable">
+          <div class="messageImage valign-wrapper">
+            <img class="circle" src="http://placehold.it/120">
+          </div>
+          <div >
+            <div class="card-content">
+
+              <!-- Replace this with dynamic address data -->
+              <p>This is a sample message that came from this user. This may have a question or a response about the home</p>
+
+
+            </div>
+
+            <div class="card-content right-align">
+              <!-- Replace this with the number of bids that have been made -->
+              <p>3/1/18</p>
+            </div>
+          </div>
+        </div>
+      </div>
+     </div>
+
+    </div>
+ </div>
+</div>
```

Comment:
I think these views are really good. Next steps to add routes and functionalitity.